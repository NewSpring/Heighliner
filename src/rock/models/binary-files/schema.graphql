
type BinaryFile implements Node {
  id: ID!
}