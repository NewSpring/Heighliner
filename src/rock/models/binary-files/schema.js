export default [
  `
  type BinaryFile implements Node {
    id: ID!
  }
`,
];
