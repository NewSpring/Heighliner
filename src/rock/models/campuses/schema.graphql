

type Location implements Node {
  id: ID!
  name: String
  street1: String
  street2: String
  city: String
  state: String
  country: String
  zip: String
  latitude: Float
  longitude: Float
  distance: Float
}

type Campus implements Node {
  id: ID!
  entityId: Int!
  name: String
  shortCode: String
  guid: String
  services: [String]
  url: String
  locationId: ID
  location: Location
}
