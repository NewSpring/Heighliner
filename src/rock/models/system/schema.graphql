
type DefinedValue implements Node {
  id: ID!
  value: String
  description: String
}