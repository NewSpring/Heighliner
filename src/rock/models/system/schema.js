
export default [`
  type DefinedValue implements Node {
    id: ID!
    _id: Int
    value: String
    description: String
  }

  type Note implements Node {
    id: ID!
    entityId: Int
    text: String
  }
`];
