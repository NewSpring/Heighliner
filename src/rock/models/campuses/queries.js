
export default [
  "campuses(id: Int, name: String): [Campus]",
];
