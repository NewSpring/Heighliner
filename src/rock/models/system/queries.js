
export default [
  `definedValues(
    id: Int!,
    limit: Int = 20,
    skip: Int = 0,
    all: Boolean = false,
  ): [DefinedValue]`,

  `notes(
    limit: Int = 20,
    skip: Int = 0,
    types: [String],
  ): [Note]`,
];
