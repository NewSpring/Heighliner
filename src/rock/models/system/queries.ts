
export default [
  `definedValues(
    id: Int!,
    limit: Int = 20,
    skip: Int = 0
  ): [DefinedValue]`,
];
