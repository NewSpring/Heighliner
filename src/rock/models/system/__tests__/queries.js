import queries from "../queries";
import schema from "../schema";

it("matches snapshot", () => {
  expect(queries).toMatchSnapshot();
});

it("matches schema snapshot", () => {
  expect(schema).toMatchSnapshot();
});
