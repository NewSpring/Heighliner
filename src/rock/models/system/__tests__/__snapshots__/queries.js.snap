exports[`test matches schema snapshot 1`] = `
Array [
  "
  type DefinedValue implements Node {
    id: ID!
    _id: Int
    value: String
    description: String
  }

  type Note implements Node {
    id: ID!
    entityId: Int
    text: String
  }
",
]
`;

exports[`test matches snapshot 1`] = `
Array [
  "definedValues(
    id: Int!,
    limit: Int = 20,
    skip: Int = 0,
    all: Boolean = false,
  ): [DefinedValue]",
  "notes(
    limit: Int = 20,
    skip: Int = 0,
    types: [String],
  ): [Note]",
]
`;
